sentry-android-core/src/main/java/io/sentry/android/core/AndroidOptionsInitializer.java

```diff
@@ -56,4 +54,9 @@ private static void createsEnvelopeDirPath(SentryOptions options, Context contex
     }
     options.setCacheDirPath(envelopesDir.getAbsolutePath());
   }
+
+  private static void addProcessors(SentryOptions options, Context context) {
+    options.addEventProcessor(new MainEventProcessor(options));
```

Comment:
This one needs to go inside `SentryOptions` so that it's added even if used outside of Android.